batch-models/src/main/scala/org/ekstep/analytics/job/report/StateAdminReportHelper.scala

```diff
@@ -30,16 +30,17 @@ trait StateAdminReportHelper extends  BaseReportsJob {
       .withColumn("explodedlocation", explode(when(size(col("locationids")).equalTo(0), array(lit(null).cast("string")))
         .otherwise(when(col("locationids").isNotNull, col("locationids"))
           .otherwise(array(lit(null).cast("string"))))))
-
     val subOrgJoinedDF = subOrgDF
-      .where(col("status").equalTo(1) && not(col("isrootorg")))
+      .where(col("status").equalTo(1))
```

Comment:
if you have removed condition, can you rename var? It is misleading.

Reply:
changed to activeSuborgDF